packages/@react-native-windows/cli/package.json

```diff
@@ -34,7 +34,7 @@
     "uuid": "^3.3.2",
     "xml-formatter": "^2.4.0",
     "xml-parser": "^1.2.1",
-    "xmldom": "^0.5.0",
+    "xmldom": "github:xmldom/xmldom#0.7.0",
```

Comment:
Can we just use the @xmldom/xmldom package instead of grabbing this from Github? I think this versioning may confuse things like component governance in the future, or some package managers.